src/reactNativeUtils.js

```diff
@@ -101,3 +101,60 @@ export function androidRelease(): string {
   // (If changing the implementation, adjust comment below jsdoc.)
   return (Platform.constants.Release: string);
 }
+
+/**
+ * The manufacturer of the Android device.
+ *
+ * E.g., "Google", "samsung".
+ */
+// This is Build.MANUFACTURER as of RN v0.68.7:
+//   https://reactnative.dev/docs/platform#constants
+//   https://github.com/facebook/react-native/blob/v0.68.7/ReactAndroid/src/main/java/com/facebook/react/modules/systeminfo/AndroidInfoModule.java#L73
+//   https://developer.android.com/reference/android/os/Build#MANUFACTURER
+export function androidManufacturer(): string {
+  invariant(Platform.OS === 'android', 'androidRelease called on iOS');
+
+  // Flow isn't refining `Platform` to a type that corresponds to values
+  // we'll see on Android.
+  //
+  // (If changing the implementation, adjust comment below jsdoc.)
```

Comment:
Then this last comment, and the blank line, can be left out too — because now the implementation just below this is just a couple of lines away from that other comment, so there's no longer a need for a second comment to point to it.